eWonConnector-gateway/src/main/java/org/imdc/ewon/EwonUtil.java

```diff
@@ -178,6 +185,9 @@ public static String httpGet(String url) throws Exception {
 
          // Generate a "cleaned" exception based on the response code.
          if (httpResponse == HTTP_UNAUTHORIZED) {
+            logger.error("Verify your account name, password, and developer ID are correctly"
+                  + " entered on the Ewon Connector configuration page.");
+            logger.error("Ewon account credentials are invalid!");
```

Comment:
I feel like this should say Talk2M account credentials instead of Ewon account credentials.

Reply:
Should we change the "Ewon Account Information" on the config page to say "Talk2M Account Information"?

Reply:
Yeah, I think so. It just seems confusing to have Ewon Account information and Ewon Device information, when they're both sections for account information

Reply:
Okay, ill make that change